DatadogCore/Sources/Kronos/KronosTimeStorage.swift

```diff
@@ -31,7 +31,7 @@ internal enum KronosTimeStoragePolicy {
 
 /// Handles saving and retrieving instances of `KronosTimeFreeze` for quick retrieval
 internal struct KronosTimeStorage {
-    private var userDefaults: UserDefaults
+    private var userDefaults: UserDefaults // swiftlint:disable:this required_reason_api_name
```

Comment:
💡 This is true positive - `UserDefaults` is the only Required Reason API we use. [It is declared in our manifest](https://github.com/DataDog/dd-sdk-ios/blob/973fb6eb1820faca459e1c14fe4391284ca8290d/DatadogCore/Resources/PrivacyInfo.xcprivacy#L24).